packages/fx-core/resource/package.nls.json

```diff
@@ -129,6 +129,7 @@
   "core.projectVersionChecker.vscodeUseNewVersion": "The current project is incompatible with the installed Teams Toolkit, please install %s or above and reopen this project.",
   "core.projectVersionChecker.cliUseNewVersion": "[core] Your TeamsFx CLI is too old to support current project, please upgrade to the latest version using command below:\nnpm install -g @microsoft/teamsfx-cli@latest",
   "core.projectVersionChecker.incompatibleProject": "The current project is incompatible with the installed version of Teams Toolkit.",
+  "core.projectVersionChecker.vs.incompatibleProject": "The project in the solution is created with Teams Toolkit preview feature - Teams App Configuration Improvements. You can turn on the preview feature to continue.",
```

Comment:
This message is used in vs already. Update here to keep aligned

Reply:
can you follow the key pattern rule: error.migration.{sub-category}.{Error name capitalized first letter}

Reply:
Here is a warning message, not error message. Will follow this format in new error message